src/super_gradients/scripts/visualize_and_save_results_from_directory.py

```diff
@@ -131,6 +138,19 @@ def main(
             cv2.imwrite(output_path, image_output)
             print(f"Saved in: {output_path}.")
 
+            if output_json:
+                dict_output = {
+                    "class_names": output.class_names,
+                    "bboxes_xyxy": output.prediction.bboxes_xyxy.tolist(),
+                    "confidence": output.prediction.confidence.tolist(),
+                    "labels": output.prediction.labels.tolist(),
+                    "image_shape": output.prediction.image_shape,
+                }
+                with open(output_path.removesuffix(".png") + ".json", "w") as file:
+                    json.dump(dict_output, file)
+
+                print(f"Saved in: {output_path.removesuffix('.png')}.json.")
```

Comment:
small suggestion to remove duplication:
```suggestion
                out_json_path = output_path.with_suffix(".json")
                with open(out_json_path, "w") as file:
                    json.dump(dict_output, file)

                print(f"Saved in: {out_json_path}")
```